.github/workflows/release.yml

```diff
@@ -0,0 +1,64 @@
+name: Build & Release
+
+on:
+  release:
+    types: [ published ]
```

Comment:
What does this mean? How do I trigger this workflow? I usually want to publish to PyPI on all tags.

Reply:
I misread what you said earlier to be honest. So this workflow would be triggered whenever you publish a new release here on GitHub. But i just noticed, that you don't do that 😄 
![image](https://github.com/eerimoq/bitstruct/assets/52598363/adf40ffd-259f-4506-9a12-d3efe9555bdd)

I'll change that back so it is triggered by tags.

Reply:
I used the solution from [here](https://github.com/orgs/community/discussions/25302#discussioncomment-3247370)
Hopefully that works.

Reply:
Let's try it out. I'll merge!